src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -532,6 +538,7 @@
     <Reference Include="System.Reflection.Primitives" />
     <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
```

Comment:
This adding this dependency fine for System.Private.Xml?

Reply:
Nothing inbox can depend on System.Runtime.CompilerServices.Unsafe right now because of https://github.com/dotnet/corefx/pull/25959

Reply:
So I should probably be able to switch to just use `src/mscorlib/shared/Internal/Runtime/CompilerServices/Unsafe.cs`.

Reply:
ok, just realized that Unsafe.cs is not meant to be consumed by corefx...

Reply:
It comes with its own set of issues: You would need to convert all contacts reference to source references in all affected projects. It won't be easy to do for tests - the tests like to be built against contracts.